timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -1077,24 +1077,37 @@ template <typename RealType> void declare_nonbonded_interaction_group(py::module
     std::string pyclass_name = std::string("NonbondedInteractionGroup_") + typestr;
     py::class_<Class, std::shared_ptr<Class>, timemachine::Potential>(
         m, pyclass_name.c_str(), py::buffer_protocol(), py::dynamic_attr())
-        .def("set_atom_idxs", &timemachine::NonbondedInteractionGroup<RealType>::set_atom_idxs, py::arg("atom_idxs"))
+        .def(
+            "set_atom_idxs",
+            &timemachine::NonbondedInteractionGroup<RealType>::set_atom_idxs,
+            py::arg("atom_idxs"),
+            py::arg("col_atom_idxs") = py::none())
```

Comment:
maybe just break API and rename `atom_idxs` to `row_atom_idxs`?

Reply:
I'm actually in favor of `row` and `col` here still, because the underlying implementation is literally a square symmetric `NxN` matrix with distinct `rows` and `cols`, everything internally (including neighborlists), explicitly refer to `rows` and `cols` with different semantics, eg. `row` idxs are contiguous in ordering and `col` idxs. We've never used `group_1` or `group_2` in any other context before, and I'd be hesitant to introduce new notation.

Reply:
Anyways - I can see pros and cons of each @jkausrelay I'll let you decide.

Reply:
- Decided to go with row/col as that is consistent with the rest of the code (NC/NR, etc)